src/components/ebay-toast/index.marko

```diff
@@ -0,0 +1,18 @@
+import processHtmlAttributes from "../../common/html-attributes"
+
+<ebay-dialog-base
+    ...input
+    open=input.open
+    baseEl="aside"
+    transitionEl="root"
+    noTrap
```

Comment:
Can we call this `modeless` instead? A modeless dialog will never trap keyboard or screenreader.

Reply:
Or just `modal=false` (default: true)